great_expectations/expectations/metrics/map_metric.py

```diff
@@ -950,31 +966,32 @@ def _sqlalchemy_column_map_condition_value_counts(
     (selectable, _, _,) = execution_engine.get_compute_domain(
         compute_domain_kwargs, domain_type="identity"
     )
+
     if "column" not in accessor_domain_kwargs:
         raise ValueError(
             "_sqlalchemy_column_map_condition_value_counts requires a column in accessor_domain_kwargs"
         )
-    column = sa.column(accessor_domain_kwargs["column"])
-    try:
-        return execution_engine.engine.execute(
-            sa.select([column, sa.func.count(column)])
-            .select_from(selectable)
-            .where(unexpected_condition)
-            .group_by(column)
-        ).fetchall()
-    except OperationalError as oe:
-        exception_message: str = "An SQL execution Exception occurred.  "
-        exception_traceback: str = traceback.format_exc()
-        exception_message += (
-            f'{type(oe).__name__}: "{str(oe)}".  Traceback: "{exception_traceback}".'
+
+    column_name = accessor_domain_kwargs["column"]
+
+    if column_name not in metrics["table.columns"]:
+        raise ge_exceptions.ExecutionEngineError(
+            message=f'Error: The column "{column_name}" in BatchData does not exist.'
         )
-        logger.error(exception_message)
-        raise ge_exceptions.ExecutionEngineError(message=exception_message)
```

Comment:
Do we not want to catch this?

Reply:
This is caught properly in the caller(s).